mailpoet/lib/Automation/Engine/Migrations/Migrator.php

```diff
@@ -0,0 +1,82 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Engine\Migrations;
+
+use MailPoet\Automation\Engine\Exceptions;
+use wpdb;
+
+class Migrator {
+  /** @var string */
+  private $prefix;
+
+  /** @var wpdb */
+  private $wpdb;
+
+  public function __construct() {
+    global $wpdb;
+    $this->prefix = $wpdb->prefix . 'mailpoet_automation_';
+    $this->wpdb = $wpdb;
+  }
+
+  public function createSchema(): void {
+    $this->runQuery("
+      CREATE TABLE {$this->prefix}workflows (
+        id int(11) unsigned NOT NULL AUTO_INCREMENT,
+        name varchar(255) NOT NULL,
+        status varchar(255) NOT NULL,
+        created_at timestamp NOT NULL,
+        updated_at timestamp NOT NULL,
+        deleted_at timestamp NULL,
+        trigger_keys longtext NOT NULL,
+        steps longtext,
+        PRIMARY KEY (id)
+      );
+    ");
+
+    $this->runQuery("
+      CREATE TABLE {$this->prefix}workflow_runs (
+        id int(11) unsigned NOT NULL AUTO_INCREMENT,
+        workflow_id int(11) unsigned NOT NULL,
+        trigger_key int(11) unsigned NOT NULL,
```

Comment:
There is a type mismatch here, `trigger_key` is a `String` in the `WorkflowRun` class.

Reply:
@brezocordero Good catch! Checking what's stored in my DB now and there's just `0`. This column is not used anywhere yet but I thought it would be good to store which trigger started a workflow. I'll fix it.